Makefile

```diff
@@ -533,6 +584,8 @@ daikon.tar daikon.zip: doc-all $(DOC_PATHS) $(EDG_FILES) $(README_PATHS) $(DAIKO
 	mkdir ${TMPDIR}/daikon
 
 	mkdir ${TMPDIR}/daikon/doc
+	# dummy www directory to remove end user Make warnings
```

Comment:
Should we include the www directory in the release instead?  It's not very big, and doing so would eliminate a difference between the repository and the release.